docs/tutorials/quickstart.md

```diff
@@ -17,6 +24,7 @@ way to start it is to run it via a container:
     podman run -it --rm --net host rmohr/activemq:5.15.9-alpine
```

Comment:
Should: Also remove this reference to activemq

Reply:
I have removed activemq reference, a couple of things I notice while doing so which are the side effects of removing the Storm default:

1.  The cli will forget/reload the env every time so
 [`blueapi controller run count '{"detectors": ["current_det", "image_det"]}`](https://github.com/DiamondLightSource/blueapi/blob/24c7e35820cdcfb31b9876ef7d9798590a363dec/docs/how-to/run-cli.md?plain=1#L33)
in the quickstart guide will not work unless we set the env with #649 or do 
 ` blueapi -c src/script/stomp_config.yml controller run count '{"detectors": ["current_det", "image_det"]}'`
2. `blueapi serve` will technically work but there will be half a page of warning ever 10 secs.

Reply:
Point 1 is addressed with #674